client/resource_cluster.go

```diff
@@ -53,6 +53,21 @@ func ResourceCluster() *schema.Resource {
 				MaxItems: 1,
 				Elem:     sparkServerSchema(),
 			},
+			"labels": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Labels to attach to the object",
+
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"attribute": {
```

Comment:
shouldn't this be plural?

Reply:
When one uses lists of resource schemas in terraform they're written more like this:
```
attribute {
  key = "country"
  value = "australia"
}

attribute {
  key = "state"
  value = "nsw"
}
```

So no. I I think it makes more sense to use singluar form here, because each is specified separately.

Reply:
oh yes, got to love terraform's random syntax.